File: 2017 - 2/Programação de Computadores/delano prova/CopiaRV/src/removeVogais/RemoveVogais.java
package removeVogais;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;

public class RemoveVogais {

    public void removeVogais(String fileIn, String fileOut) {
        int i;

        try {
            FileReader fr = new FileReader(fileIn);
            BufferedReader input = new BufferedReader(fr);

            FileWriter fw = new FileWriter(fileOut);
            BufferedWriter output = new BufferedWriter(fw);

            while ((i = fr.read()) != -1) {
                char c = (char) i;

                switch (c) {
                    case 'A':
                    case 'E':
                    case 'I':
                    case 'O':
                    case 'U':
                    case 'a':
                    case 'e':
                    case 'i':
                    case 'o':
                    case 'u':
                    case ' ': break;
                    default:
                    {
                        fw.write(c);
                        break;
                    }
                }                
            }

            input.close();
            
            output.close();
        } catch (IOException e) {

        }
    }

    public static void main(String[] args) {
        RemoveVogais copia = new RemoveVogais();
        copia.removeVogais("entrada.txt", "saida.txt");
    }
}

File: 2017 - 2/Programação de Computadores/delano prova/CopiaRemoveVogais/RemoveVogais.cpp
#include "RemoveVogais.h"

#include <iostream>
#include <fstream>
#include <string>

using namespace std;

RemoveVogais::RemoveVogais() {
}

RemoveVogais::~RemoveVogais() {
}

void RemoveVogais::copia(string fileIn, string fileOut) {
    string line;
    ifstream input(fileIn.c_str());
    ofstream output;
    output.open(fileOut.c_str());

    if (input.is_open()) {
        while (getline(input, line)) {
            for (int i = 0; i < line.length(); i++) {
                char c = line.at(i);
                switch (c) {
                    case 'A':
                    case 'E':
                    case 'I':
                    case 'O':
                    case 'U':
                    case 'a':
                    case 'e':
                    case 'i':
                    case 'o':
                    case 'u':
                    case ' ': break;
                    default:
                    {
                        output << c;
                        break;
                    }
                }
            }
            output << endl;
        }
    }

    input.close();
    output.close();
}
